modules/ROOT/pages/message-logging-policy.adoc

```diff
@@ -63,7 +64,27 @@ Error handler after API flow:
 
 image::apim-message-logging-policy-states-3.png[State after the error handler executes]
 
-== Logging Policy in XML
+
+== Configuring Policy Parameters
+
+When you apply the Message Logging policy to your API from the UI, you can configure the following parameters:
+
+image::apim-message-logging-policy-ui.png[Message Logging Policy UI]
+
+[%header%autowidth.spread]
+|===
+|Parameter |Description
+|Message |A DataWeave expression that's transformed into a message to be logged.
+Note: If the payload is used as part of the message log, do not configure the Listener as non-repeatable (by default this setting is off).
+|Conditional |The condition expressed as a DataWeave expression that returns true or false that needs to be satisfied to log the message. This field is optional.
+|Category |The category that the log has. This parameter is optional. If no category is set, the default is the `org.mule.runtime.logging.policy-<policy id>` value.
+|Level |Defines if the message is logged according to the configuration in your log4j2 file.
```

Comment:
The levels values are the same as the ones used by log4j, but they are not in the log4j file. Both levels and file are two different things in this case.

Reply:
Done! Thanks!